kafka-ui-api/src/main/java/com/provectus/kafka/ui/serde/RecordSerDe.java

```diff
@@ -13,6 +14,9 @@
   class DeserializedKeyValue {
     @Nullable String key;
     @Nullable String value;
+    @Nullable MessageFormat format;
+    @Nullable Integer valueSize;
```

Comment:
let's also add keySize and headersSize

Reply:
Will do

Reply:
@germanosin  I will add keySize. Are you sure about headers size? It will require some refactoring because we use key-value pair everywhere. If we need headers size anyway - it's going to be sum of key sizes and value sizes for all headers, right?

Reply:
Yes, this is for all headers keys & values

Reply:
Done